### PR TITLE
Use python super function instead of calling by class name

### PR DESCRIPTION
**Pull request by vishichoudhary**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [X] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

### CODE REVIEW

File: picard/ui/infodialog.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def get_type_widget(self, type_text):
 class InfoDialog(PicardDialog):
 
     def __init__(self, obj, parent=None):
-        PicardDialog.__init__(self, parent)
+        sueper().__init__(parent)
```
Comment (zas) [inline picard/ui/infodialog.py]: ^^typo

Comment (vishichoudhary) [reply picard/ui/infodialog.py]: oops. sorry. will fix it in next commit.

**Comment by vishichoudhary** [comment]
@zas i think it's complete.

**Comment by zas** [comment]
@vishichoudhary it looks ok to me, can you please do an interactive rebase and fix the typo in the commit it belongs, keeping only working commits ?

**Comment by vishichoudhary** [comment]
@zas ok i will do.

**Pull request merged**